net.sourceforge.pmd.eclipse.plugin/src/main/java/net/sourceforge/pmd/eclipse/runtime/properties/IProjectProperties.java

```diff
@@ -72,13 +72,13 @@
     /**
      * @return Returns the project Rule Set.
      */
-    RuleSet getProjectRuleSet() throws PropertiesException;
+    RuleSets getProjectRuleSets() throws PropertiesException;
```

Comment:
Please keep the old ones for backwards compatibility and add the methods, that deal with RuleSets